sites/all/modules/ding_unilogin/lib/DingUniloginServiceBase.class.inc

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * @file
+ * Base class for classes communicating with the UNI•Login services.
+ */
+
+/**
+ * Base class for UNI•Login services.
+ */
+class DingUniloginServiceBase {
```

Comment:
This call should probably be declared abstract. It can not/should not be instantiated.